src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -680,14 +685,20 @@ class WalletService(
                             servicesWithUpdateFailed[serviceName] = serviceGlobalStatus
                         }
 
-                        Pair(servicesUpdatedSuccessfully, servicesWithUpdateFailed)
+                        Triple(
+                            servicesUpdatedSuccessfully,
+                            servicesWithUpdateFailed,
+                            updatedApplications
+                        )
                     }
-                    .map { (servicesUpdatedSuccessfully, servicesWithUpdateFailed) ->
+                    .map {
+                        (servicesUpdatedSuccessfully, servicesWithUpdateFailed, updatedApplications)
+                        ->
                         WalletServiceUpdateData(
                             servicesUpdatedSuccessfully,
                             servicesWithUpdateFailed,
                             wallet.copy(
-                                applications = walletApplications.values.toList(),
+                                applications = updatedApplications.values.toList(),
```

Comment:
what if there are applications not present in the updated list?
I guess they're lost, is it really wanted?

Reply:
You are right, fixed with 0447535 :)